PerfectDay/Controllers/TaskController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using PerfectDay.Repositories;
+using PerfectDay.Entities;
+using AutoMapper;
+
+namespace PerfectDay.Controllers
+{
+    [Route("api/[controller]")]
+    public class TaskController : ControllerBase
```

Comment:
No need to use ControllerBase

Reply:
This is needed for Http stutus codes responds